core/src/main/java/tc/oc/pgm/command/MapCommand.java

```diff
@@ -208,6 +208,12 @@ public void map(Audience audience, CommandSender sender, @Text MapInfo map) {
               .build());
     }
 
+    audience.sendMessage(
+        TextComponent.builder()
+            .append(mapInfoLabel("map.info.license"))
+            .append(TextComponent.of(map.getLicense().getFullName(), TextColor.GOLD))
```

Comment:
Can we get a hover click to view the license? Also can we make the license more human-readable / understandable? Tags like "Commercial" or something?

Assume the user doesn't know much about CC.

Reply:
Yes, great ideas!

Reply:
I tried changing them a little, let me know if this is what you were thinking. I think keeping the words as they are in the license is logical because if you google "Attribute license" the CC license will appear.